rswag-specs/lib/rswag/specs/openapi_formatter.rb

```diff
@@ -158,6 +129,120 @@ def remove_invalid_operation_keys!(value)
         value.delete(:produces) if value[:produces]
         value.delete(:request_examples) if value[:request_examples]
       end
+
+      def parse_parameters(doc)
+        doc[:paths]&.each_pair do |_k, path|
+          path.each_pair do |_verb, endpoint|
+            is_hash = endpoint.is_a?(Hash)
+            if is_hash && endpoint[:parameters]
+              mime_list = endpoint[:consumes] || doc[:consumes]
+              parse_endpoint(endpoint, mime_list) if mime_list
+            end
+            remove_invalid_operation_keys!(endpoint)
+          end
+        end
+      end
+
+      def parse_endpoint(endpoint, mime_list)
+        parameters = endpoint[:parameters]
+        # There can only be 1 body parameter in Swagger 2.0, so while in OAS3 we interpret
+        # body parameters as formData, only consider the first body we encounter.
+        schema_param = parameters.find { |p| p[:schema] if p[:in] == :body }
+        parse_body_parameter(endpoint, schema_param, mime_list) if schema_param
```

Comment:
@stevegeek, the Swagger 2.0 support is being dropped in this branch.
It seems to me that the `parse_body_parameter` function can be dropped.